src/components/inputs/Radio.tsx

```diff
@@ -96,6 +98,7 @@ const Radio = React.forwardRef<HTMLDivElement, RadioProps>(function RadioFn(
 		padding = { bottom: 'small' },
 		size = 'medium',
 		iconColor = 'gray0',
+		value,
```

Comment:
It's used in the RadioGroup to update the component state and to set the Radio "checked" prop

Reply:
So, I've thought about this, I'm going to approve this one because with current implementation I can't think of a clean way to have a value. But I'm also going to open a issue to improve the internal implementation of the radio component